PanModal/Presentable/PanModalPresentable.swift

```diff
@@ -85,6 +85,15 @@ public protocol PanModalPresentable: AnyObject {
     */
     var transitionAnimationOptions: UIView.AnimationOptions { get }
 
+    /**
+    The background view color.
+
+    - Note: This is only utilized at the very start of the transition.
+
+    Default Value is black.
+    */
+    var backgroundColor: UIColor { get }
```

Comment:
Could we rename this to something along the lines of `panModalBackgroundColor` - so it doesn't conflict with `UIView`'s `backgroundColor` property